smtp-crawler/src/main/java/be/dnsbelgium/mercator/smtp/domain/crawler/SmtpConversationCache.java

```diff
@@ -0,0 +1,26 @@
+package be.dnsbelgium.mercator.smtp.domain.crawler;
+
+import be.dnsbelgium.mercator.smtp.dto.SmtpConversation;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+@Service
+public class SmtpConversationCache {
```

Comment:
Is there any cache invalidation? For example, if a pod of this would be around for two months, will there be a second visit to the same mail server? I can imagine that they might have new features supported over time.